DFImageManager/Source/Core/Caching/DFImageCache.m

```diff
@@ -52,7 +55,7 @@ - (instancetype)init {
 - (nullable DFCachedImageResponse *)cachedImageResponseForKey:(nullable id<NSCopying>)key {
     DFCachedImageResponse *response = [_cache objectForKey:key];
     if (response) {
-        if (response.expirationDate > CACurrentMediaTime()) {
+        if (response.expirationDate > mach_absolute_time()) {
```

Comment:
Yea the reasoning here is that CACurrentMediaTime is not available on the watch. Any suggestions for a replacement works.